containers/Makefile

```diff
@@ -21,30 +20,41 @@ ifndef CONCENT_DIR
 endif
 $(info CONCENT_DIR=$(CONCENT_DIR))
 
+ifndef IMAGE_VERSION
+        IMAGE_VERSION := $(shell git describe --always)
+endif
+$(info IMAGE_VERSION=$(IMAGE_VERSION))
+
+ifndef IMAGE_PREFIX
+        IMAGE_PREFIX :=
+endif
+$(info IMAGE_PREFIX=$(IMAGE_PREFIX))
+
+
 all: build/ nginx-proxy concent-api postgresql
 
 nginx-proxy: $(NGINX_PROXY_SOURCE) build/nginx-proxy/concent-api-assets.tar build/nginx-proxy/concent-api-docs.tar
-	docker build -t nginx-proxy:$(VERSION) build/nginx-proxy/
-	docker tag nginx-proxy:$(VERSION) nginx-proxy:latest
+	docker build -t $(IMAGE_PREFIX)nginx-proxy:$(IMAGE_VERSION) build/nginx-proxy/
+	docker tag $(IMAGE_PREFIX)nginx-proxy:$(IMAGE_VERSION) $(IMAGE_PREFIX)nginx-proxy:$(IMAGE_VERSION)
```

Comment:
This is wrong again. Now with `IMAGE_PREFIX=abc-` and `IMAGE_VERSION=xyz` your image will be tagged only as `abc-nginx:xyz` and the second line is pointless (it does not create a new tag).

The expected result is to get two tags: `abc-nginx:xyz` and `abc-nginx:latest`.